change/@fluentui-web-components-fe7f57a9-65bb-4816-9f01-0aded96e8ce9.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Added additional export in component.",
```

Comment:
```suggestion
  "comment": "feat(menu-item): export MenuItemRole constants",
```

Reply:
Is there a convention for change file comments posted somewhere?